File: Dynamic Programming/jumpingStairs.py
def jump(num):
  dp=[0]*(num+1)
  dp[0]=1
  dp[1]=1
  for i in range(2,num+1):
    dp[i]=dp[i-1]+dp[i-2]
  return dp[num]  
  
print(jump(3))
  

File: Dynamic Programming/jumpingStairs.js
function jump(num){
  if( num ==1 )return 1;
  const dp = new Array(num).fill(-1)
  dp[0]=1
  dp[1]=1
  for(let i=2;i<=num+1;i++)
    dp[i]=dp[i-1]+dp[i-2]
  return dp[num] 
}
  
console.log(jump(5));

  